app/src/ui/pull-request-quick-view.tsx

```diff
@@ -121,6 +125,40 @@ export class PullRequestQuickView extends React.Component<
     return { top: clamp(alignedMiddle, minTop, maxTop) }
   }
 
+  private getPointerPosition(
+    position: React.CSSProperties | undefined,
+    quickViewHeight: number
+  ): React.CSSProperties | undefined {
+    // 23 = half a pr list item.
+    const defaultTop = { top: 20 }
+    if (position === undefined) {
+      return defaultTop
+    }
+
+    const { top, bottom } = position
+    const quickViewTopZero = this.getTopPRList()
+    const prListItemTopWRTQuickViewTopZero =
+      this.props.pullRequestItemTop - quickViewTopZero
+
+    let quickViewTop
+    if (top !== undefined) {
+      quickViewTop = parseInt(top.toString())
+    } else if (bottom !== undefined) {
+      const bottomAsNum = parseInt(bottom.toString())
+      quickViewTop = bottomAsNum - quickViewHeight + quickViewTopZero
+    }
```

Comment:
This seems unused? I.e. there doesn't seem to be a case where bottom can be set?

Reply:
I had like 3-4 prs this was built on top of and I had tons of merge conflicts... and I appear to be left with an older version of this method.  Sorry for the confusion/not really ready pr. (I used to use a top/bottom and then found a simpler solution)